src/data/checks.js

```diff
@@ -150,6 +150,28 @@ function bindFindArtistOrAlias(boundFind) {
   };
 }
 
+function bindFindThing(boundFind) {
+  return unknownRef => {
+    // Find findable Things in the order defined in keylessReplacerOrder
+    for (const boundFindFunc of keylessReplacerOrder.map(key => boundFind[key])) {
+      const thing = boundFindFunc(unknownRef, {mode: 'quiet'});
```

Comment:
If you're going to iterate directly over `keylessReplacerOrder` up earlier...

https://github.com/hsmusic/hsmusic-wiki/blob/b65249c1bafbae77d8f8b4a3cc4656994f43e440/src/content/dependencies/transformContent.js#L91-L92

...can you do the same here, too?

```suggestion
    for (const key of keylessReplacerOrder) {
      const thing = boundFind[key](unknownRef, {mode: 'quiet'});
```